doc/JSON_INFO.md

```diff
@@ -2522,35 +2521,35 @@ at level `2` to the item.
 
 ### Traits/Mutations
 
-See MUTATIONS.md
+See [MUTATIONS.md](MUTATIONS.md)
 
 ### Trait Migrations
 
-See MUTATIONS.md
+See [MUTATIONS.md](MUTATIONS.md)
 
 ### Traps
 
 ```C++
     "type": "trap",
-    "id": "tr_beartrap",  // Unique ID
-    "name": "bear trap",  // In-game name displayed
+    "id": "tr_beartrap", // Unique ID
+    "name": "bear trap", // In-game name displayed
     "color": "blue",
     "symbol": "^",
-    "visibility": 2,  // 1 to ??, affects detection
-    "avoidance": 7,  // 0 to ??, affects avoidance
-    "difficulty": 3,  // 0 to ??, difficulty of assembly & disassembly
-    "trap_radius": 1,  // 0 to ??, trap radius
-    "action": "blade",
+    "visibility": 2, // 0 to infinity, 0 means a blatantly obvious trap, the higher, the harder to spot.
+    "avoidance": 7, // 0 to infinity, affects how easy it is to dodge a triggered trap. 0 means dead easy, the higher the harder.
+    "difficulty": 3, // 0 to 99, 0 means disarming is always successful (e.g funnels or other benign traps), 99 means disarming is impossible.
+    "trap_radius": 1, // 0 to infinity, radius of space the trap needs when being deployed.
+    "action": "blade", // C++ function that gets run when trap is triggered, usually in trapfunc.cpp
     "map_regen": "microlab_shifting_hall",  // a valid overmap id, for map_regen action traps
-    "benign": true,
-    "always_invisible": true,
-    "funnel_radius": 200,  // millimeters?
-    "comfort": 4,
+    "benign": true, // For things such as rollmats, funnels etc. They can not be triggered.
+    "always_invisible": true, // Super well hidden traps the player can never detect
+    "funnel_radius": 200, // millimeters. The higher the more rain it will capture.
+    "comfort": 0,
```

Comment:
```suggestion
    "comfort": 0, // Level of sleeping comfort this trap provides. The higher the more easily character can fall asleep on this tile.
```

Reply:
I omitted this on purpose because it is common to furniture, terrain and traps and I hate duplicating stuff. But yes, the section is called "Common to furniture and terrain" and does not include traps.  I just implicitely referenced the existing documentation for this and I hope this does the job.